mailpoet/assets/js/src/form_editor/translations.ts

```diff
@@ -1,11 +1,11 @@
-import { setLocaleData } from '@wordpress/i18n';
+import { LocaleData, setLocaleData } from '@wordpress/i18n';
 
-export default (moduleTranslations): void => {
+export default (moduleTranslations:string[]): void => {
   moduleTranslations.forEach((translations) => {
     const parsed = JSON.parse(translations);
     if (!parsed || !parsed.locale_data?.messages) {
       return;
     }
-    setLocaleData(parsed.locale_data.messages);
+    setLocaleData(<LocaleData>parsed.locale_data.messages);
```

Comment:
Same here, I think we should stick to `as`.

Reply:
Addressed in 60f94f6ea